frontend/packages/console-app/src/components/pdb/PodDisruptionBudgetField.tsx

```diff
@@ -23,24 +23,29 @@ export const PodDisruptionBudgetField: React.FC<PodDisruptionBudgetFieldProps> =
   });
   const pdb = getPDBResource(pdbResources, obj);
```

Comment:
> This doesn't need to be addressed as part of this PR, but we might have another bug here. It seems possible that more than one PDB could match the current obj. Shouldn't we list all of them instead of only the first match we find?

Right, it is possible for a resource to have more than one PDB applied if the user bypassed the target resource pages which menu actions state is Edit PDB, and use PDB creation page or CLI to apply it, but I am not how K8 handle this type of scenario.

Reply:
> > This doesn't need to be addressed as part of this PR, but we might have another bug here. It seems possible that more than one PDB could match the current obj. Shouldn't we list all of them instead of only the first match we find?
> 
> Right, it is possible for a resource to have more than one PDB applied if the user bypassed the target resource pages which menu actions state is Edit PDB, and use PDB creation page or CLI to apply it, but I am not sure how K8 handle this type of scenario.